package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "diacritics-transliterator",
+  "version": "0.1.0",
+  "description": "Diacritic transliteration tools using diacritics.io API",
+  "main": "index.js",
+  "scripts": {
+    "test": "ava test/*.js",
+    "linting": "./node_modules/.bin/eslint './*.js'"
```

Comment:
The path doesn't seem to be necessary. `"linting": "eslint './*.js'"` seems to be enough, like you've done this with `ava`. It's documented [here](https://docs.npmjs.com/misc/scripts#path). I didn't know this too.